nekoyume/Assets/_Scripts/State/LocalLayerModifier.cs

```diff
@@ -379,16 +357,16 @@ public static void AddReceivableQuest(Address avatarAddress, int id)
         /// Change the equipment's mounting status.
         /// </summary>
         /// <param name="avatarAddress"></param>
-        /// <param name="itemId"></param>
+        /// <param name="nonFungibleId"></param>
         /// <param name="equip"></param>
         /// <param name="resetState"></param>
         public static void SetItemEquip(
             Address avatarAddress,
-            Guid itemId,
+            Guid nonFungibleId,
             bool equip,
-            bool resetState = true)
+            bool resetState = default)
```

Comment:
I know the bool default value is false, is this okay?

Reply:
Yes it is. I have fixed all other arguments.